docs/components/card.md

```diff
@@ -1,275 +1,179 @@
 # Card
 
-**KCard** is meant to highlight content in grids or group relative information together.
+KCard is a styled container that organizes together related content and actions.
 
-<KCard title="Card Title" body="Body Content"/>
-
-```html
-<KCard title="Card Title" body="Body Content"/>
-```
-
-## Props
-
-### title
-
-String to be used in the title slot.
-
-<KCard title="Title">
-  <template v-slot:body>
-    I am the body.
+<KCard>
+  <template #title>
+    Card Title
   </template>
-</KCard>
-
-```html
-<KCard title="Title">
-  <template v-slot:body>
-    I am the body.
+  <template #actions>
+    <KDropdown
+      :items="[
+        { label: 'Props', to: { path: '/components/card#props' } },
+        { label: 'Slots', to: { path: '/components/card#slots' } }
+      ]"
+    >
+      <KButton
+        appearance="tertiary"
+        class="icon-button"
+        size="small"
+      >
+        <MoreIcon />
```

Comment:
question: should the more icon be "built-in" somehow if it's going to be standard?